CenterForOpenScience/osf.io
# [ENG-2530] Improve email template language

## Purpose

This is meant to improve the email temple language for adding contributors to draft registrations, but in recreating the email locally I discovered some additional clean-up was necessary.

## Changes

- changes template language
- renamed preprint related email templates to remove `_brand` suffix because they were being used as part of a unused naming scheme
- removed unused `mimetype='html'` kwarg from `send_mail` this is obsolete as our mail is now exclusively html
- clean-up `notify_added_contributor` there were some readability issues and inexplicable choices in this function I have fixed.

## QA Notes

Please make verification statements inspired by your code and what your code touches.
- Verify the email send with the correct language.

## Documentation



## Side Effects

None that I know of.

## Ticket

https://openscience.atlassian.net/browse/ENG-2530
